src/frontend/src/views/Upload/components/Metadata/components/Table/index.tsx

```diff
@@ -1,4 +1,5 @@
 import { Table as MuiTable, TableBody, TableHead } from "@material-ui/core";
+import { reduce } from "lodash";
```

Comment:
I think we might need `"lodash/reduce"` to prevent importing the whole `lodash` library here!

[source](https://dev.to/icokie/lodash-import-done-right-290n)